README.md

```diff
@@ -154,7 +154,7 @@ The following table lists the global, default, and other parameters supported by
 | `global.image.username` | Global Docker image registry username | `nil` |
 | `global.image.password` | Global Docker image registry username | `password` |
 | `global.image.tag` | Global Docker image registry tag | `mender-3.6.2` |
-| `global.imagePullSecrets` | Global Docker registry secret names as an array | `[]` (does not add image pull secrets to deployed pods)  |
+| `global.image.externalImagePullSecrets` | Optional list of existing Image Pull Secrets in the format of `- name: my-custom-secret` | `[]` |
```

Comment:
I actually prefer the old (unused) name :see_no_evil: 
The `external` part seems redundant and "image" is repeated in the value key.

Reply:
> I actually prefer the old (unused) name 🙈 The `external` part seems redundant and "image" is repeated in the value key.

The thing is that `imagePullSecrets` is already taken:
* `global.imagePullSecrets`: unused
* `imagePullSecret`: its a template defined [here](https://github.com/mendersoftware/mender-helm/blob/master/mender/templates/_image-pull-secret.yaml)

What I want to do is to create this new ImagePullSecret, same as for HM, and don't interfere with the current one, created in with the `imagePullSecret` template.
Maybe something like `global.image.additionalPullSecrets` ?

Reply:
I just found that `global.imagePullSecrets` is also inherited to subcharts. E.g. redis:
```
      imagePullSecrets:
        - name: map[name:docker-hub-secret-rollnrenew-robtest]
```
So I'm gonna move it to the `default` instead.